<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mywidget</class>
 <widget class="QWidget" name="mywidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>startX:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>startY:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_startX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_startY">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>intervalY:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_intervalX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_intervalY">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>intervalX:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>nodeAttr:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_nodeAttr">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_routeAttr">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>routeAttr:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_finish">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>finish</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_cols">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>rows:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>cols:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tx_rows">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
